etl/steps/data/garden/energy/2023-12-12/primary_energy_consumption.meta.yml

```diff
@@ -43,26 +44,31 @@ tables:
         title: Primary energy consumption (TWh)
         short_unit: TWh
         unit: terawatt-hours
-        description: Primary energy consumption, measured in terawatt-hours per year.
+        description_short: Measured in terawatt-hours.
```

Comment:
I think it'd be useful to have a note in `description_key` on what terawatt-hours unit for all of the relevant indicators -e.g., `A terawatt-hour (TWh) is a unit of energy equivalent to one trillion watts being used for one hour, commonly used to measure large-scale electricity production and consumption.`